src/main/scala/org/broadinstitute/dsde/agora/server/model/AgoraApiJsonSupport.scala

```diff
@@ -167,7 +174,19 @@ object AgoraApiJsonSupport extends DefaultJsonProtocol {
 
   implicit val AccessControlPairFormat = jsonFormat3(EntityAccessControl)
 
-  implicit object AgoraStatusFormat extends RootJsonFormat[AgoraStatus] {
+  implicit val MethodRepoMethodFormat = jsonFormat3(MethodRepoMethod)
+
+  // AttributeString is just a case class with one String in it, but for some reason jsonFormat1 is not sufficient
```

Comment:
`jsonFormat1` would translate `AttributeString("this.hello")` as `{ "value": "this.hello" }` but we want simply `"this.hello"`